src/firebase/constants.js

```diff
@@ -6,6 +6,8 @@ export const firestoreCollections = {
 export const storageBuckets = {
   SIE_RAW_IMGS: 'sie-raw-images',
   SIE_STIMULI_IMGS: 'sie-stimuli-images',
+  SIE_USER_SELECTIONS: 'sie-user-selections',
+  SIE_CI_IMAGES: 'sie-ci-images',
```

Comment:
Was checking firebase storage and I confirmed that there are these two buckets ready to be used - 'sie-ci-images' and 'sie-user-selections'. 

Just wanted to double check - is 'SIE_CI_IMAGES' supposed to be used in this function?

Reply:
Thanks for the double-check! No this is supposed to be used for fetching the generated CI image, which is about to be developed later today :)